keystrokes/src/config/constants.js

```diff
@@ -8,6 +8,23 @@ const config = {
 };
 firebase.initializeApp(config);
 
+export const characterDetails = {
+	race: ["", "Night Elf", "Human", "Worgen", "Toren", "Blood Elf", "Orc", "Troll", "Undead", "Dwarf", "Gnome", "Goblin"],
+	class: {
+		"": [],
```

Comment:
What's with the empty values for class and race?

Reply:
spec is dependent on what class you pick 
`{characterDetails.class[characterClass].map(spec` so i added an empty class so that it would work, then just added an empty race so it looked more consistent in the buttons. Maybe there is a better way to do it though.